changelog.md

```diff
@@ -26,3 +26,7 @@
 
 ## 0.5.2
 - Fix `RegisterModule(Mock<T> mock)` by removing `new()` restriction
+
+## Unreleased
+- Added `MockContainerBuilder`.
```

Comment:
```suggestion
- Added `ContainerBuilderMockExtension`.
```

Reply:
Addressed in: https://github.com/messerli-informatik-ag/test-utility/pull/20/commits/4d33a0c474230aebf9a54c1d4d870ecb7c356783